tests/ondemand/ondemand_json_pointer_tests.cpp

```diff
@@ -0,0 +1,50 @@
+#include "simdjson.h"
+#include "test_ondemand.h"
+
+using namespace simdjson;
+
+namespace json_pointer_tests {
+    bool easytest() {
+        TEST_START();
+        ondemand::parser parser;
+        auto cars_json = R"( [
+        { "make": "Toyota", "model": "Camry",  "year": 2018, "tire_pressure": [ 40.1, 39.9, 37.7, 40.4 ] },
+        { "make": "Kia",    "model": "Soul",   "year": 2012, "tire_pressure": [ 30.1, 31.0, 28.6, 28.7 ] },
+        { "make": "Toyota", "model": "Tercel", "year": 1999, "tire_pressure": [ 29.8, 30.0, 30.2, 30.5 ] }
+        ] )"_padded;
+        auto cars = parser.iterate(cars_json);
+        double x = cars.at_pointer("/0/tire_pressure/1");
```

Comment:
This can throw an exception, which is fine but we sometimes compile simdjson with exceptions disabled which is going to create trouble.  You can do an exceptionless variant with something that looks like this...

```
double x;
ASSERT_SUCCESS(cars.at_pointer("/0/tire_pressure/1").get(x));
```

You can also keep your code as-is, but guard it with macro checks like this: https://github.com/simdjson/simdjson/blob/40cba172ed66584cf670c98202ed474a316667e3/tests/ondemand/ondemand_active_tests.cpp#L8

Reply:
The story is that C++ is a language with exceptions but, sometimes, people want to do away with exceptions. I am told that game developers do not want to use exceptions. So we have this bastard language (C++ without exceptions) that we try to support.

Reply:
I see...